applications/FluidDynamicsApplication/python_scripts/flows_measuring_process.py

```diff
@@ -0,0 +1,16 @@
+# Importing the Kratos Library
```

Comment:
Following the convention of the application, this file should be named `apply_flows_measuring_process.py`. This would simplify the clean up a lot as soon as we finish the register of the processes.